src/varefaz/layerstoolkit.json

```diff
@@ -0,0 +1,44 @@
+{
+    "title": "Layers Toolkit - RRO Layers",
+    "description": "Layers Toolkit has been created by me, Varefaz, and Piereligio. Our intent is to make more and more mods easily installable and removable, without the need of reinstall the whole system if you want to remove them. For any problem you encounter, contact us here: varefaz@gmail.com or piereligio@gmail.com .",
+    "author": "Varefaz",
+    "link": "https://play.google.com/store/apps/details?id=com.varefaz.LayersToolkit",
+    "backup_link": false,
+    "icon": "https://drive.google.com/open?id=0BwMVe_WsmvSUUHFpOFBVVmRrM28",
```

Comment:
So how can i do to have the app on the showcase now?
Il 15 mar 2016 3:57 PM, "Denis Suarez Jr" <notifications@github.com> ha
scritto:

> In src/varefaz/layerstoolkit.json
> <https://github.com/BitSyko/layers_showcase_json/pull/28#discussion_r56176734>
> :
>
> > @@ -0,0 +1,44 @@
> > +{
> > +    "title": "Layers Toolkit - RRO Layers",
> > +    "description": "Layers Toolkit has been created by me, Varefaz, and Piereligio. Our intent is to make more and more mods easily installable and removable, without the need of reinstall the whole system if you want to remove them. For any problem you encounter, contact us here: varefaz@gmail.com or piereligio@gmail.com .",
> > +    "author": "Varefaz",
> > +    "link": "https://play.google.com/store/apps/details?id=com.varefaz.LayersToolkit",
> > +    "backup_link": false,
> > +    "icon": "https://drive.google.com/open?id=0BwMVe_WsmvSUUHFpOFBVVmRrM28",
>
> Don't have permission to view the images on google drive and guessing
> there not direct links to the images. Also this layers has been accepted
> for the new database
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub:
> https://github.com/BitSyko/layers_showcase_json/pull/28/files#r56176734
>